grails-app/controllers/org/broadinstitute/orsp/SubmissionController.groovy

```diff
@@ -21,6 +22,16 @@ class SubmissionController extends AuthenticatedController {
                 ])
     }
 
+    def getSubmissions() {
+        Issue issue = queryService.findByKey(params.id)
+        if (issueIsForbidden(issue)) {
+            redirect(controller: 'Index', action: 'index')
```

Comment:
We should never be redirecting inside a controller. The error should be returned and the UI needs to deal with it.